firebase-datatransport/src/main/java/com/google/firebase/datatransport/TransportRegistrar.java

```diff
@@ -22,21 +22,23 @@
 import com.google.firebase.components.Component;
 import com.google.firebase.components.ComponentRegistrar;
 import com.google.firebase.components.Dependency;
-import java.util.Collections;
+import com.google.firebase.platforminfo.LibraryVersionComponent;
+import java.util.Arrays;
 import java.util.List;
 
 @Keep
 public class TransportRegistrar implements ComponentRegistrar {
   @Override
   public List<Component<?>> getComponents() {
-    return Collections.singletonList(
+    return Arrays.asList(
         Component.builder(TransportFactory.class)
             .add(Dependency.required(Context.class))
             .factory(
                 c -> {
                   TransportRuntime.initialize(c.get(Context.class));
                   return TransportRuntime.getInstance().newFactory(CCTDestination.LEGACY_INSTANCE);
                 })
-            .build());
+            .build(),
+        LibraryVersionComponent.create("fire-transport", BuildConfig.VERSION_NAME));
```

Comment:
Is this related to this pull request?

Reply:
Not exactly, but it makes us to measure how long firelog takes to start, will keep it for now.